src/en/news/blog/2022/dashboard-rbd-scaling/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Dashboard scability of RBD images"
+date: "2022-09-01"
+author: "Pere Diaz Bou"
+image: "images/image.jpg"
+tags:
+  - "mgr"
+  - "performance"
+  - "python"
+  - "rbd"
+---
+
+## Overview
+
+Management of RBD images in the Dashboard module has been challenging due to its low performant
+behaviour with a high enough number of images. With enough images tables would see long loading
+times which might seem infinite rendering them unusable.
+
+After introducing server side pagination with RBD images we've been able to see a clear and expected
+**reduction of around 99%** in latency.
+
```

Comment:
Images speaks louder than words. What about some charts of the before & after? When did the dashboard become unusable due to the number of RBD images? What's the average time per image? What about a recording of the before & after?

Reply:
i was wondering if we should add that data there tbh since the purpose of this was to share that the RBD page is now usable, not a technical only post. Regarding the recordings I can do that.

Reply:
In my experience, arguments are more compelling when properly seasoned.